src/main/java/com/frameworkium/lite/ui/listeners/CaptureListener.java

```diff
@@ -59,13 +59,19 @@ private void sendFinalScreenshot(ITestResult result, String action) {
 
         var thrw = result.getThrowable();
         var driver = UITestLifecycle.get().getWebDriver();
-        if (null != thrw) {
-            takeScreenshotAndSend(action, driver, thrw);
-        } else {
-            var command = new Command(action, "n/a", "n/a");
-            takeScreenshotAndSend(command, driver);
-        }
-    }
+		//		# if getting web driver returns null because test has been skipped
+		//        do not send final screenshot
+		if (driver == null && action.equalsIgnoreCase("skip")) {
+			return;
+		} else {
+			if (null != thrw) {
+				takeScreenshotAndSend(action, driver, thrw);
+			} else {
+				var command = new Command(action, "n/a", "n/a");
+				takeScreenshotAndSend(command, driver);
+			}
+		}
+	}
```

Comment:
Something like this might be even safer?

```suggestion
        if (driver == null) {
            return;
        }
        if (null != thrw) {
             takeScreenshotAndSend(action, driver, thrw);
         } else {
             var command = new Command(action, "n/a", "n/a");
             takeScreenshotAndSend(command, driver);
         }
    }
```